### DISCUSSION TITLE
[@types/react] throwing exception for `size` attribute of `<hr>`

### DISCUSSION THREAD

**Discussion by corocoto** [DefinitelyTyped/DefinitelyTyped#65780] [Issues with a @types package] [answered]
## Description
I tried to use native  `size` attribute for `<hr>` tag. But at the moment, I saw that typescript throwing an exception for this block of code.

> P.S. TS exception:
> `error TS2322: Type '{ size: number; }' is not assignable to type 'DetailedHTMLProps<HTMLAttributes<HTMLHRElement>, HTMLHRElement>'.
  Property 'size' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLHRElement>, HTMLHRElement>'.`

## Steps to reproduce:
1. Create basic react application
2. Create test component which will render `hr` tag with set `size` attribute:
```jsx
function TestComponent() {
  return (
    <hr size={1}  />
  );
}
```
3. Run `tsc -p tsconfig.json --noEmit` from root directory of your project

## Expected Behavior
Don't have any exceptions from typescript side.

## Environment:
* `typescript` version: 5.1.3
* `@types/react` version: 18.2.12
*  `@types/react-dom` version: 18.2.5

[a comment by an automated account is not shown]

**Comment by corocoto** [comment]
But if supporting of this attribute has been removed by it's deprecated status (please see ["Browser compatibility" section](https://developer.mozilla.org/ru/docs/Web/HTML/Element/hr#browser_compatibility)) - I understand it, and will close the discussion.

**Reply by Semigradsky** [reply, marked as answer]
Also attribute was totally removed from HTML spec
